lib/src/Base/Optim/NLopt.cxx

```diff
@@ -348,6 +348,7 @@ void NLopt::run()
     result.store(inP, outP, absoluteError, relativeError, residualError, constraintError);
   }
 
+  result.setIterationNumber(size);
```

Comment:
That's not a really number of iteration but the number of evaluations.

Reply:
IIRC I already requested to merge these numbers, maybe we can agree now?

See how they are used in OT, it does not make sense to define two numbers.

Reply:
I agree one could use only one var.

But note that TNC uses getMaximumIterationNumber whereas NLopt uses getMaximumEvaluationNumber as maximum number of evaluations used by their solver.

Reply:
and Cobyla uses getMaximumIterationNumber() to define the maximum number of evaluations ;-)

Reply:
So maybe NLopt could use getMaximumIterationNumber too, and we deprecated get/setMaximumIterationNumber, update the doc of getMaximumIterationNumber to mention it also applies to the number of evaluations depending on the solver, what do you think ?

Reply:
Ok I guess it can be part of another PR